static/locales/en-US.yaml

```diff
@@ -0,0 +1,314 @@
+
+    'If you enjoy using FreeTube, consider donating via Liberapay or through our Bitcoin address.': >
+    'If you enjoy using FreeTube, consider donating via Liberapay or through our Bitcoin address.'
+
+    'BTC: 1Lih7Ho5gnxb1CwPD4o59ss78pwo2T91eS': 'BTC: 1Lih7Ho5gnxb1CwPD4o59ss78pwo2T91eS'
+
+#On Channel Page
+Subscriber: Subscriber
+Subscribers: Subscribers
+Subscribe: Subscribe
+Unsubscribe: Unsubscribe
+Video: Video
+#& Videos
+    #& Sort By
+        Newest: Newest
+        Oldest: Oldest
+        #& Most Popular
+#& Playlists
+    #& About
+    Channel Description: Channel Description
+    Featured Channels: Featured Channels
+    Search Channel: Search Channel
+    View Full Playlist: View Full Playlist
+
+# On Channel Playlist Page
+Share Playlist: Share Playlist
+    Copy YouTube Link: Copy YouTube Link
+    Open in YouTube: Open in YouTube
+    Copy Invidious Link: Copy Invidious Link
+    Open in Invidious: Open in Invidious
+
+# On Video Watch Page
+#* Published
+#& Views
+Change Format: Change Format
+    Use Dash Formats: Use Dash Formats
+    Use Legacy Formats: Use Legacy Formats
+    Use Audio Formats: Use Audio Formats
+Share: Share
+    Copy Link: Copy Link
+    Open Link: Open Link
+    Copy Embed: Copy Embed
+    Open Embed: Open Embed
+    # On Click
+    URL has been copied to the clipboard: URL copied to clipboard
+Mini Player: Mini Player
+Click to View Comments: Click to View Comments
+Show Comments: Show Comments
+Up Next: Up Next
+
+# Toast Messages
+Local API Error (Click to copy): Local API Error (Click to copy)
+Falling back to Invidious API: Falling back to Invidious API
+Falling back to the local API: Falling back to the local API
+Subscriptions have not yet been implemented: Subscriptions have not yet been implemented
+Loop is now disabled: Loop is now disabled
+Loop is now enabled: Loop is now enabled
+Shuffle is now disabled: Shuffle is now disabled
+Shuffle is now enabled: Shuffle is now enabled
+Playing Next Video: Playing Next Video
+Playing previous video: Playing previous video
+Playing next video in 5 seconds.  Click to cancel: Playing next video in 5 seconds.  Click to cancel
+Canceled next video autoplay: Canceled next video autoplay
+
+# Currently Not Added from Live
+    # On Settings Page
+        # General Settings
+            Default Landing Page: Default Landing Page
+                #& Subscriptions
+                #& Trending
+                #& Most Popular
+                #& Playlists
+                #& History
+            Region for Trending: Region for Trending
+                #! List countries
+        # Player Settings
+            Remember History: Remember History
+            Autoplay Videos: Autoplay Videos
+            Autoplay Playlists: Autoplay Playlists
+            Play Next Video: Play Next Video
+            Turn on Subtitles by Default: Turn on Subtitles by Default
+            Grab videos locally: Grab videos locally
```

Comment:
This can be removed as it's replaced by `Force Local backend for legacy formats`